build-custom-image/workflow.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     - default: Dockerfile
       description: The relative path of the Dockerfile in the build context.
       name: dockerfile
-    - default: 't9kpublic/kaniko-project-executor:latest'
+    - default: 't9kpublic/kaniko-project-executor:debug'
```

Comment:
why using this `debug` tag?

Reply:
switch to image t9kpublic/kaniko-project-executor:debug because it has /bin/sh